src/main/kotlin/no/nav/syfo/infrastructure/database/repository/PSenOppfolgingKandidat.kt

```diff
@@ -0,0 +1,31 @@
+package no.nav.syfo.infrastructure.database.repository
+
+import no.nav.syfo.domain.OnskerOppfolging
+import no.nav.syfo.domain.Personident
+import no.nav.syfo.domain.SenOppfolgingKandidat
+import no.nav.syfo.domain.SenOppfolgingSvar
+import java.time.OffsetDateTime
+import java.util.*
+
+data class PSenOppfolgingKandidat(
+    val id: Int,
+    val uuid: UUID,
+    val createdAt: OffsetDateTime,
+    val updatedAt: OffsetDateTime,
+    val personident: Personident,
+    val varselAt: OffsetDateTime,
+    val svarAt: OffsetDateTime?,
+    val onskerOppfolging: String?,
```

Comment:
Burde denne vær en enum? 😊
```suggestion
    val onskerOppfolging: EnEllerAnnenEnum?,
```

Reply:
Tanken var å bare bruke String i P-objektet og mappe til enum når vi mapper til domain, men det blir litt hipp som happ for meg :man_shrugging:

Reply:
Allright, hva enn som blir lettest og tydeligst 🤌